sdk/src/kalix.ts

```diff
@@ -605,22 +599,54 @@ export class Kalix {
     process.exit(0);
   }
 
+  /** @internal */
+  reportError(error: discovery.UserFunctionError): void {
+    const msg = this.reportErrorLogic(
+      error.code,
+      error.message,
+      error.detail,
+      error.sourceLocations,
+      error.severity,
+    );
+    switch (error.severity) {
+      case discovery.Severity.INFO:
+      case discovery.Severity.WARNING:
+        console.log(msg);
```

Comment:
There is a `console.warn` but it's an alias for `console.error`, which prints to stderr. This puts warnings on stdout with `console.log` instead, as GCP logging will mark stderr as ERRORs. But maybe having it use `console.warn` and print to stderr is better anyway?